### PR TITLE
add unixodbc-dev to testing docker image

### PR DESCRIPTION
**Pull request by kwigley**
### Description

Updates Dockerfile for `fishtownanalytics/test-container` image. Adds `unixodbc-dev` packages required for installing `pyodbc` ([docs](https://github.com/mkleehammer/pyodbc/wiki/Install#debian-stretch)).

### Checklist
 - [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
 - [ ] I have run this code in development and it appears to resolve the stated issue
 - [ ] This PR includes tests, or tests are not required/relevant for this PR
 - [x] I have updated the `CHANGELOG.md` and added information about my change to the "dbt next" section.

### CODE REVIEW

**Comment by kwigley** [comment]
Once approved, I'll build and push an image with tag `fishtownanalytics/test-container:10` to use in dbt-spark

File: Dockerfile [hunk 1]
```diff
@@ -2,26 +2,49 @@ FROM ubuntu:18.04
...
+    software-properties-common \
+    make \
+    build-essential \
+    ca-certificates \
+    libpq-dev \
+    libsasl2-dev \
+    libsasl2-2 \
+    libsasl2-modules-gssapi-mit \
+    libyaml-dev \
+    unixodbc-dev \
```
Comment (kwigley) [inline Dockerfile:22]: sorry for formatting! this is the only new package here

**Review by gshank — APPROVED** [review]

**Review by drewbanin — APPROVED** [review]
LGTM! Were you able to test this locally?

**Comment by kwigley** [comment]
> LGTM! Were you able to test this locally?

Yep 👍

**Pull request merged**